Sources/CloudEnvironment/PushSDKCredentials.swift

```diff
@@ -22,12 +22,12 @@ import Foundation
 public class PushSDKCredentials {
 
   public let appGuid      : String
-  public let appSecret    : String
+  public let apiKey       : String
```

Comment:
This is SemVer Major, of course.

Reply:
Yes - `9.0.0`

Reply:
We released `8.0.0` when Watson migrated to IAM credentials in June, and made similar changes for those services.